doc/content/verification_and_validation/ver-1hb.md

```diff
@@ -0,0 +1,64 @@
+# ver-1hb
+
+# Equilibrating Enclosures
+
+## Problem set up
+
+This verification problem models two species '$T_2$' (gaseous tritium) and '$D_2$' (gaseous deuterium) equilibrating in pressure between two enclosures. It recreates the ver-1hb case in [!cite](ambrosek2008verification). Enclosure 1 is pre-charged with tritium and enclosure 2 is pre-charged with deuterium. The gases flow between the two enclosures until the $T_2$ and $D_2$ partial pressures in the two enclosures equalibrate. The pressure change rates for this system for gas $T_2$ are given by
+\begin{equation} \label{eq:dt_P1_T}
+\frac{dP^1_{T_2}}{dt} = \frac{Q}{V} (P^2_{T_2} - P^1_{T_2}),
+\end{equation}
+\begin{equation} \label{eq:dt_P2_T}
+\frac{dP^2_{T_2}}{dt} = \frac{Q}{V} (P^2_{T_2} - P^1_{T_2}),
+\end{equation}
+and for gas $D_2$ are given by
+\begin{equation} \label{eq:dt_P1_D}
+\frac{dP^1_{D_2}}{dt} = \frac{Q}{V} (P^2_{D_2} - P^1_{D_2}),
+\end{equation}
+\begin{equation} \label{eq:dt_P2_D}
+\frac{dP^2_{D_2}}{dt} = \frac{Q}{V} (P^2_{D_2} - P^1_{D_2}),
+\end{equation}
+where $Q$ is the volumetric flow rate (m$^3$s$^-1$), $V$ is the volume (m$^3$), $P^i_j$ is the pressure in enclosure $i$ of gas species $j$ ($j$ = $T_2$ or $D_2$).
```

Comment:
```suggestion
where $Q$ is the volumetric flow rate (m$^3 \cdot$s$^{-1}$), $V$ is the volume (m$^3$), $P^i_j$ is the pressure in enclosure $i$ of gas species $j$ ($j$ = $T_2$ or $D_2$).
```

Reply:
Added manually due to some conflicts merging multiple commits on the same line.